proxy/build.gradle

```diff
@@ -18,6 +18,20 @@ task buildProxyImage(dependsOn: deployJar, type: Exec) {
   commandLine 'docker', 'build', '-t', 'proxy', '.'
 }
 
+task deployProxy(dependsOn: [buildProxyImage, ':verifyDeployment']) {
+  doLast {
```

Comment:
Is there any way to put these in to a single exec() command? Would that be the correct call?